geodiff/src/geodiff.cpp

```diff
@@ -29,6 +29,11 @@
 #include <string>
 #include <vector>
 
+#include "json.hpp"
+
+using json = nlohmann::json;
```

Comment:
I am not big fan of `using ...` since for example then you cannott have local variable names called "json" and use use `j` instead. ...

but if @wonder-sk is happy I am easy :)